README.md

```diff
@@ -63,7 +63,9 @@ These are the options. Most of them are set with environment variables.
   -o, --device-ownership-code <deviceOwnershipCode>  PIN/ownership code of the device (default: "123456")
   -m, --mqtt-messages-prefix <mqttMessagesPrefix>    The prefix for the MQTT for this tenant for sending and receiving device messages (default: "")
   -v, --verbose                                      Output debug information
-  -t, --app-type <appType>                           Type of device application. "mss" or "atv2" (default: "atv2")
+  -t, --app-type <appType>                           Specifies the shadow to use. For custom shadow, pass a JSON-encoded shadow object or relative path to json file. Otherwise,
+                                                     pass "mss" or "atv2" to automatically generate a conformal shadow
+                                                     (default: "atv2")                           
```

Comment:
The `cli.ts` shows (correctly?) there's no default, so does this README need to be corrected next time you're doing a PR?